package.json

```diff
@@ -19,17 +19,13 @@
   ],
   "peerDependencies": {
     "babel-eslint": "^7.2.3",
-    "eslint": "^3.17.1",
-    "eslint-config-airbnb": "^14.1.0",
-    "eslint-plugin-import": "^2.2.0",
-    "eslint-plugin-jsx-a11y": "^4.0.0",
-    "eslint-plugin-react": "^6.10.0"
-  },
-  "devDependencies": {
-    "np": "^2.10.0"
+    "eslint": "^3.19.0",
```

Comment:
They've not - the diff is awkward because I removed `np` from the dev dependencies (as it doesn't really make sense to include it).